src/main/java/com/car/app/singular/ICarService.java

```diff
@@ -1,4 +1,4 @@
-package com.car.app.service;
+package com.car.app.singular;
```

Comment:
The package name should be controller not singular :laughing:

Reply:
Am I wrong ? haha

![image](https://user-images.githubusercontent.com/78147936/107396686-46620b00-6afe-11eb-9c33-f9cf8f18bea0.png)

Reply:
hahaha, yes you're right. What I meant was to rename to package to singular name instead of plural, i.e. controllers -> controller

Reply:
hahaha. okay. I'll